gamemodes/terrortown/entities/entities/ttt_game_text.lua

```diff
@@ -39,7 +39,7 @@ function ENT:KeyValue(key, value)
 		self.Receiver = tonumber(value)
 
 		if not self.Receiver or self.Receiver < 0 or self.Receiver > 5 then
-			ErrorNoHalt("ERROR: ttt_game_text has invalid inputReceiver value\n")
+			ErrorNoHaltWithStack("ERROR: ttt_game_text has invalid inputReceiver value\n")
```

Comment:
same here for these two. I'm not saying this is wrong, I just wonder if it adds any meaningful information here?